### DISCUSSION TITLE
keep getting logged out from google on individual containers

### DISCUSSION THREAD

**Discussion by Admond124** [mozilla/multi-account-containers#2707] [Q&A] [answered]
hello
im having a problem that i keep getting logged out of my google accounts on the individual containers im using.
for some context, ive been using i use different google accounts in different containers and was never really logged out. Until recently where ive been gettin logged out..like the cookies just disappear and it logs me out. It really affects me because i use google signin alot.

these are the extensions i have if necessary..i dont know , i have a feeling its ublock origin thats causing it.
![image](https://github.com/user-attachments/assets/941c07bc-8c4a-45b9-801c-8e671737d08a)

please has anyone had a similar issue and is there a solution if yes?

**Comment by achernyakevich-sc** [comment]
@Admond124 As far as know Google logout user every two weeks and ask to enter password again. At least it behave this way for my Google accounts.

**Reply by Admond124** [reply]
mine is every few hours it happens...after i close firefox for a while then come back im logged out across all google accounts.

**Reply by achernyakevich-sc** [reply]
It looks like you have turned on Firefox setting `Delete cookies and site data when Firefox is closed`. Could you open about:preferences#privacy and check that the checkbox on screenshot below is not ticked.

![image](https://github.com/user-attachments/assets/56da9672-e301-4520-a374-afe778c5888f)

**Reply by Admond124** [reply]
i just checked and it wasnt turned on and never was.
![{56EF286B-CA99-4F45-9100-3138DF592B19}](https://github.com/user-attachments/assets/a85f30de-4ecb-4d9a-b9df-a23a494480a1)

I do think its ublock origin, so id temporarily disable that and loginto the accounts. Appreciate the responses >>

**Reply by achernyakevich-sc** [reply]
> Appreciate the responses

Please, share results of your research. It would be valuable to other users of MAC. :)

**Reply by Admond124** [reply, marked as answer]
so yh, seems it was ublock origin that was causing the problems.
i whitelisted accounts.google.com entirely and since then i havent been logged out ..atleast as of right now.
sorry for the delays haha.
new year wishes :)

**Reply by Olghn** [reply]
This seems it may be related to an issue I am having. Sometimes I want to confine a Google related website to a 'Google' container but I am finding I cannot do so. These sites seem to escape the container - the tab reverts to a general one without the colouring indicating it is my 'Google' container.
This applies when pasting the URL into a container tab or when right-clicking a tab and telling it to open the site in the 'Google' container.
e.g.
https://www.genetichomeland.com/consentmgt.asp?fc=alwaysshow

[I tried disabling uBlock origin but no change]
